src/non-profits-contact.html

```diff
@@ -2,31 +2,45 @@
 layout: default
 title: Non-Profits Contact | Women Hack for Non-Profits
 ---
-
 <section class="smaller-banner">
   <div class="content">
     <h1 class="title-heading">Non-Profits Contact Form</h1>
   </div>
 </section>
-<section>
+<section class="col-lg-3 col-md-7 col-sm-9 col-xs-12 position central">
```

Comment:
is there a screenshot perhaps of what it looks like now under different break points?